tests/gtest/sock/sock_socket.cc

```diff
@@ -45,7 +45,7 @@ class sock_socket : public sock_base {};
  *    Create UDP socket
  * @details
  */
-TEST_F(sock_socket, ti_1) {
+TEST_F(sock_socket, DISABLED_ti_1) {
```

Comment:
please avoid disabling tests for all targets. You can use `sys_rootuser` if it is specific for **root**